Balances faq update (#70)
wealthkernel/api-documentation-guides: docs/balances/FAQs.md

Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.

> Valuations are dated, and only transactions with a timestamp on or before a valuation's date will be reflected in the valuation.
> 
> Balances are not dated and are continuously updated; therefore, they will reflect any transactions booked to the portfolio as they occur.
> 
> <!-- theme: info -->
> > An existing dated valuation will be recalculated if a transaction is booked with a timestamp of the date of the valuation.
> 
> The cash balance is continuously updated so shows any action, e.g. a deposit, immediately, making the full amount available for withdrawal or trading straight away.
> 

Reviewer: ```suggestion
Valuations are dated, and only transactions with a timestamp on or before a valuation's date will be reflected in the valuation.

Balances are not dated and are continuously updated; therefore, they will reflect any transactions booked to the portfolio as they occur.
```

Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.

> Valuations are created at approximately the same time every day, for the previous day. Transactions booked with a date of `t` are only reflected in the valuation for `t+1`.

Reviewer: ```suggestion
Valuations are dated, and only transactions with a timestamp on or before a valuation's date will be reflected in the valuation.
```

This level of detail around when valuations are created etc. should be in a Valuations section of the docs IMO.

> The balance is not dated, and is continuously updated, and will therefore reflect any actions taken on the portfolio, such as deposits, orders, or bonuses, as they happen.

Reviewer: ```suggestion
Balances are not dated and are continuously updated; therefore, they will reflect any transactions booked to the portfolio as they occur.
```

### Example

```mermaid
gantt
dateFormat YYYY-MM-DD HH:mm
axisFormat %m-%d %H:%M
title Dated valuations
todayMarker off

    section Valuation

>     Latest Valuation 0GBP :active, val1, 2024-02-19 17:00, 13h

Reviewer: I'd be explicit with the valuation date. "Latest" is only available internally.

Reviewer: Also timestamps give the impression "Latest valuation" lives from 2024-02-19 17:00 -> 2024-02-20 06:00 when in reality it is two valuation intervals.

    Deposit 100GBP :milestone, deposit, 2024-02-20 02:00, 0h
    Valuation 2024-02-20 calculated :milestone, calc1, 2024-02-20 06:00, 0h
    Valuation 2024-02-20 100GBP :active, val1, 2024-02-20 06:00, 49h
    Bonus awarded 10GBP :milestone, bon, 2024-02-20 12:00, 0h
    Valuation 2024-02-21 calculated :milestone, calc2, 2024-02-21 06:00, 0h
    Valuation 2024-02-21 110GBP :active, val2, 2024-02-21 06:00, 25h

> ```

Reviewer: I've been thinking about these charts as they didn't look right to me but I could put my finger on why, but I think I know why now: they are showing when a valuation is readable but showing transaction events alongside, which are taken into account for a valuation even if it is not readable.

I think maybe we remove them to unblock and have a think about how we can diagram them in way that can't be misinterpreted.

Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.

> A given day's valuation is created at the approximately same time every day, and would not be updated with transactions that are created after that time. The next day's valuation would include those transactions. The balance is continuously updated with ongoing transactions as they happen, and will therefore reflect any actions taken on the portfolio, such as deposits, orders, or bonuses.

Reviewer: 🤔 This reads to me like it is saying any transaction created after a valuation was created won't be included in the valuation, which isn't the case. I think we need to be careful with the wording to make it explicit that valuations are tied to closing price and that is why they are available T+1.

> Example: 100GBP deposit occurs before the valuation for 2024-02-20 is calculated. When the valuation is calculated, the cash portion will reflect that:

Reviewer: This reads like the calculation is a one time thing.
